composer.json

```diff
@@ -4,26 +4,19 @@
     "type": "project",
     "description": "Keboola Microsoft SQL DB Writer",
     "keywords": ["keboola", "microsoft", "sql server", "mssql", "db", "writer"],
-    "authors": [
-        {
-            "name": "Miroslav Čillík",
-            "email": "miro@keboola.com"
-        }
-    ],
     "require": {
-        "php": "^7.1",
-        "ext-PDO": "*",
-        "keboola/csv": "^1.4",
-        "keboola/db-writer-common": "^5.8",
-        "keboola/retry": "^0.5.0",
-        "symfony/config": "^4.4",
-        "symfony/process": "^4.4"
+        "php": "^8.2",
+        "keboola/csv": "^4.1",
```

Comment:
Tady se to používá jen jako reader? Takže ten BC break 4.x není relevantní?

Reply:
jj ve writerech je jen Reader, protože se nikam nezapisuje, tak by to vadit nemělo